packages/react-devtools-shared/src/backend/console.js

```diff
@@ -134,68 +147,75 @@ export function patch({
         let shouldAppendWarningStack = false;
-        if (consoleSettingsRef.appendComponentStack) {
-          const lastArg = args.length > 0 ? args[args.length - 1] : null;
-          const alreadyHasComponentStack =
-            typeof lastArg === 'string' && isStringComponentStack(lastArg);
-
-          // If we are ever called with a string that already has a component stack,
-          // e.g. a React error/warning, don't append a second stack.
-          shouldAppendWarningStack = !alreadyHasComponentStack;
+        if (method !== 'log') {
+          if (consoleSettingsRef.appendComponentStack) {
+            const lastArg = args.length > 0 ? args[args.length - 1] : null;
+            const alreadyHasComponentStack =
+              typeof lastArg === 'string' && isStringComponentStack(lastArg);
+
+            // If we are ever called with a string that already has a component stack,
+            // e.g. a React error/warning, don't append a second stack.
+            shouldAppendWarningStack = !alreadyHasComponentStack;
+          }
         }
 
         const shouldShowInlineWarningsAndErrors =
           consoleSettingsRef.showInlineWarningsAndErrors &&
           (method === 'error' || method === 'warn');
 
-        if (shouldAppendWarningStack || shouldShowInlineWarningsAndErrors) {
-          // Search for the first renderer that has a current Fiber.
-          // We don't handle the edge case of stacks for more than one (e.g. interleaved renderers?)
-          // eslint-disable-next-line no-for-of-loops/no-for-of-loops
-          for (const {
-            currentDispatcherRef,
-            getCurrentFiber,
-            onErrorOrWarning,
-            workTagMap,
-          } of injectedRenderers.values()) {
-            const current: ?Fiber = getCurrentFiber();
-            if (current != null) {
-              try {
-                if (shouldShowInlineWarningsAndErrors) {
-                  // patch() is called by two places: (1) the hook and (2) the renderer backend.
-                  // The backend is what implements a message queue, so it's the only one that injects onErrorOrWarning.
-                  if (typeof onErrorOrWarning === 'function') {
-                    onErrorOrWarning(
-                      current,
-                      ((method: any): 'error' | 'warn'),
-                      // Copy args before we mutate them (e.g. adding the component stack)
-                      args.slice(),
-                    );
-                  }
-                }
+        let isInStrictMode = false;
 
-                if (shouldAppendWarningStack) {
-                  const componentStack = getStackByFiberInDevAndProd(
-                    workTagMap,
+        // Search for the first renderer that has a current Fiber.
+        // We don't handle the edge case of stacks for more than one (e.g. interleaved renderers?)
+        // eslint-disable-next-line no-for-of-loops/no-for-of-loops
```

Comment:
Looks like we use one `.eslintrc.js` file for the whole react repo, and we can't use for/of loops in other parts of the react repo because they require a Symbol polyfill for runtimes that don't support Symbols. @bvaughn for why we can disable it here. Maybe because devtools don't support these browsers?

Reply:
Extra context:

By default, all DevTools build targets share this common Babel configuration:
https://github.com/facebook/react/blob/5634ed16aaba4a507844f0502953abcf40e3a165/packages/react-devtools-shared/babel.config.js#L19-L26

Which tells Babel to target the minimum supported version of Chrome:
https://github.com/facebook/react/blob/5634ed16aaba4a507844f0502953abcf40e3a165/packages/react-devtools-extensions/chrome/manifest.json#L8

And Firefox:
https://github.com/facebook/react/blob/5634ed16aaba4a507844f0502953abcf40e3a165/packages/react-devtools-extensions/firefox/manifest.json#L10

Both of which are new enough not to require this polyfill you mention.

We override this for Hermes here by targeting IE11 (which has the same lack of support for ES6):
https://github.com/facebook/react/blob/c5cfa71948dbf4c2dd9ff64596ab663f8d559cd6/packages/react-devtools-core/webpack.backend.js#L21-L24